openwisp_monitoring/device/migrations/0009_update_device_status_for_disabled_critical_checks.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.db import migrations
+from swapper import load_model
+
+Check = load_model('check', 'Check')
+DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
+
+
+def update_device_status(apps, schema_editor):
+    critical_metrics_keys = [
+        metric['key'] for metric in settings.OPENWISP_MONITORING_CRITICAL_DEVICE_METRICS
```

Comment:
You have to access the setting, just like you are doing it here 

https://github.com/openwisp/openwisp-monitoring/pull/578/files#diff-e23325d8b8273f2f23eca24901c37542e622ff22082652b203617ea769f98ba6R465